examples/set_bucket_policy.py

```diff
@@ -1,6 +1,5 @@
 # -*- coding: utf-8 -*-
-# Minio Python Library for Amazon S3 Compatible Cloud Storage,
-# (C) 2016 Minio, Inc.
+# Minio Python Library for Amazon S3 Compatible Cloud Storage, (C) 2016 Minio, Inc.
```

Comment:
This is a license header @balamurugana we shouldn't change this because pep8 lacks understanding. It is not a correct practice.